MAUI/Maps/getting-started.md

```diff
@@ -5,6 +5,7 @@ description: Learn here about getting started with Syncfusion .NET MAUI Maps (Sf
 platform: MAUI
 control: SfMaps
 documentation: ug
+keywords: .net maui maps, .NET MAUI Maps, .net maui maps library.
```

Comment:
.net maui maps, .NET MAUI Maps - both need?

Reply:
i think we must use only small cases

Reply:
We used small cases for all md files